classes/security/authorization/internal/UserAccessibleWorkflowStageRequiredPolicy.inc.php

```diff
@@ -98,6 +97,17 @@ function _getAccessibleStageRoles($userId, $contextId, &$submission, $stageId) {
 					if(!$stageAssignments->wasEmpty()) {
 						$accessibleStageRoles[] = $roleId;
 					}
+
+					if ($roleId == ROLE_ID_SUB_EDITOR) {
+						// The requested submission must be part of their series...
```

Comment:
Series is an OMP thing right? I know it's not your code, but It was wrong when it was in ojs code, and it's still wrong here at library :P We have a general object name for series and sections right?